docs/faq.md

```diff
@@ -347,7 +347,7 @@ Using explicit public/private keys for signing is strongly recommended.
 
 <details>
 <summary>
-  <h3 style={{display:"inline-block"}}>What signing and encryption standards does NextAuth.js support?</h3>
```

Comment:
I removed this inline style as it doesn't seem to have any effect on the layout 📐